### DISCUSSION TITLE
How to disable background change on hover in VCard with "to" prop?

### DISCUSSION THREAD

**Discussion by Rusinas** [vuetifyjs/vuetify#18212] [Q&A] [answered]
https://play.vuetifyjs.com/#eNqtkc9uwjAMxl/Fy4VNIs0m4IIKYjfeYAfCIbRmdMs/JaEbQn33JW2RClw47BZ/tr+f7WzO5N3arD4imZM8oLJSBFxyDZDXVFjbPtugMDqISqPrpU4UroRCCu8XnNhfOgN7ojNOIJgoMGkkJ8u1qdGBwpx1DRfPFF6ZJqVj5mwwSwx94SobwGM4diNVyhoX4AwO99DA3hkFo7jGKJUDRF8fQPlPWKSK59EapTTwYZwsn0YvCdBZRjMyJp0bVcJmX97oeItzcuF9wnMyh1ZJWqSkmJNDCNbPGStKHdtKlFXtMo2BaavYKpYxd9ShUkhLo1aTbJJNWVn5MJQz9IrunPnx6KIJJ+MBJlT70wOovjISptkrFdIeRPbWkfoUlWLnE+oOwSI3/g51qEt06B7d7KZtuN1N6m7DRG+4buLdZaW//c3JC9+ee3MZ8l9Wb02T4zaBmzFp540TXDWS7R+HbwZr

If I specify the "to" prop in VCard, it will change color on hover. I don't need this, but I can't find the way to disable this behavior.

**Comment by ferflores507** [comment, marked as answer]
You need to add `"hover-opacity": 0,` to your custom theme `variables`.

**Reply by Rusinas** [reply]
Just in time 😂 I probably solved it differently already, but thanks!

**Reply by ferflores507** [reply]
I was going to add "I know it's a bit late" at the beggining before you responded hahaha, time has passed quickly man! You are welcome!

**Comment by kamechan001** [comment]
I found this in the source code
`.v-card:hover > .v-card__overlay {
  opacity: calc(var(--v-hover-opacity) * var(--v-theme-overlay-multiplier));
}`

you can set it in your style like this
`.v-card:hover>.v-card__overlay{
    opacity: 0;
}`

or if you use scoped you can set it like this
`:deep(.v-card:hover>.v-card__overlay) {
    opacity: 0;
}`
